docs/distributions/arch/faq.md

```diff
@@ -1,17 +0,0 @@
-# Updating Kernel
```

Comment:
this will break a link in https://wiki.t2linux.org/guides/wifi-bluetooth/#ensure-kernel-supports-otp-firmware-selection